odoo/tests/common.py

```diff
@@ -1465,7 +1547,7 @@ class HttpSavepointCase(HttpCaseCommon, SavepointCase):
 def users(*logins):
     """ Decorate a method to execute it once for each given user. """
     @decorator
-    def wrapper(func, *args, **kwargs):
+    def _users(func, *args, **kwargs):
```

Comment:
Because this makes the traceback way more readable. This is explained in the pr description 
`users.wrapper was renamed in users._users to improve traceback readability.`